cli/commands/deposit/validate.go

```diff
@@ -101,7 +98,14 @@ func validateDepositMessage(chainSpec chain.Spec) func(
 			return err
 		}
 
-		return ValidateDeposit(chainSpec, pubkey, credentials, amount, genesisValidatorRoot, signature)
+		if err = ValidateDeposit(
+			chainSpec, pubkey, credentials, amount, genesisValidatorRoot, signature,
+		); err != nil {
+			return err
+		}
+
+		cmd.Printf("✅ Deposit message is valid!")
```

Comment:
nit to add this message. Users reported not seeing anything in the validate command being confusing.